examples/portworx/getting_started/variables.tf

```diff
@@ -0,0 +1,18 @@
+# tflint-ignore: terraform_unused_declarations
+variable "cluster_name" {
+  description = "Name of cluster - used by Terratest for e2e test automation"
+  type        = string
+  default     = ""
+}
+
+variable "aws_access_key_id" {
```

Comment:
Should be marked as sensitive. Better yet remove them to use IRSA.

Reply:
About marking it as sensitive, thank you for pointing that out, I will fix that but I wanted a suggestion. After setting these as sensitive, it hides it in all the other places but when I perform a terraform destroy for the addon, the helm add-on resource that is getting deleted has a Metadata field that still outputs the plain text value for these variables 

example output :-

```# module.eks_blueprints_kubernetes_addons.module.portworx[0].module.helm_addon.helm_release.addon[0] will be destroyed
  - resource "helm_release" "addon" {
      - atomic                     = false -> null
      - chart                      = "portworx" -> null
      - cleanup_on_fail            = false -> null
      - create_namespace           = false -> null
      - dependency_update          = false -> null
      - description                = "A Helm chart for portworx" -> null
      - disable_crd_hooks          = false -> null
      - disable_openapi_validation = false -> null
      - disable_webhooks           = false -> null
      - force_update               = false -> null
      - id                         = "portworx-1h2e" -> null
      - keyring                    = "" -> null
      - lint                       = false -> null
      - max_history                = 0 -> null
      - metadata                   = [
          - {
              - app_version = ""
              - chart       = "portworx"
              - name        = "portworx-1h2e"
              - namespace   = "kube-system"
              - revision    = 2
              - values      = jsonencode(
                    {
                      - AKSInstall             = false
                      - EKSInstall             = true
                      - aut                    = false
                      - aws                    = {
                          - accessKeyId       = "plain-text-access-key-value here"
                          - eksServiceAccount = "portworx-1h2e-sa-1h2e"
                          - marketplace       = false
                          - secretAccessKey   = "plain-text-secret-key-value here"
                        }
                      - clusterName            = "portworx"
                      - clusterToken           = {
                          - create             = true
                          - secretName         = "px-vol-encryption"
                          - serviceAccountName = "px-create-cluster-token"
                        }
                      - deployCluster          = true
                      - deployOnMaster         = false
                      - deployOperator         = true
                      - disableStorageClass    = false

                    }
                )
              - version     = "2.11.0"
            },
        ] -> null
      - name                       = "portworx-1h2e" -> null
      - namespace                  = "kube-system" -> null
      - pass_credentials           = false -> null
      - recreate_pods              = false -> null
      - render_subchart_notes      = true -> null
      - replace                    = false -> null
      - repository                 = "https://raw.githubusercontent.com/portworx/eks-blueprint-helm/main/repo/stable" -> null
      - repository_cert_file       = "" -> null
      - repository_key_file        = "" -> null
      - repository_password        = (sensitive value)
      - repository_username        = "" -> null
      - reset_values               = false -> null
      - reuse_values               = false -> null
      - skip_crds                  = false -> null
      - status                     = "deployed" -> null
      - timeout                    = 1200 -> null
      - values                     = [
          - (sensitive),
        ] -> null
      - verify                     = false -> null
      - version                    = "2.11.0" -> null
      - wait                       = true -> null
      - wait_for_jobs              = false -> null

      - postrender {
          - args = [] -> null
        }
    }
```

I am not able to find a way around hiding these values in metadata. For the helm-add-on resource, look how the values field is marked sensitive but the metadata part still outputs it. Any suggestion to fix it